### ISSUE TITLE
[Bug]: mo crash by [index out of range [113] with length 113] when run tpcc 10 warehouse 50 terminals test

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#8186]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):bb28db7ff53063407672745e78cac38389e1d01c
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

In Nightly Regression test, when run tpcc 10 warehouse 500 terminials test for about serval minutes, the mo panic caused by the folloing reasons:

2023/02/28 13:22:45.492645 +0800 INFO frontend/util.go:418 time of Exec.Run : 2.81635ms connectionId 1120
2023/02/28 13:22:45.492709 +0800 INFO frontend/util.go:403 query trace status {"connection_id": 1120, "statement": "execute __mo_stmt_id_7 using @__mo_stmt_var_0", "status": "success", "span": {"trace_id": "eeb613c9-b727-11ed-ba9f-b07b25f8b524", "kind": "statement"}, "session_info": "connectionId 1120"}
2023/02/28 13:22:45.492961 +0800 INFO frontend/util.go:403 query trace {"connection_id": 1120, "query": "execute __mo_stmt_id_6 using @__mo_stmt_var_0,@__mo_stmt_var_1", "vars": "6 , 96360", "session_info": "connectionId 1120"}
2023/02/28 13:22:45.493177 +0800 INFO frontend/session.go:349 session uuid: ed85bbc3-b727-11ed-ba9f-b07b25f8b524 -> background session uuid: eeb70115-b727-11ed-ba9f-b07b25f8b524
panic: runtime error: index out of range [113] with length 113

goroutine 56598169 [running]:
github.com/matrixorigin/matrixone/pkg/common/bitmap.(*Bitmap).AddMany(...)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/common/bitmap/bitmap.go:198
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/containers.UnmarshalToMoVec({0x2d3df10, 0xc0fa4615c0})
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/containers/utils.go:89 +0x45d
github.com/matrixorigin/matrixone/pkg/vm/engine/tae/dataio/blockio.BlockRead({0x2cc5f58, 0xc144610750}, 0x0?, {0xc1643e4cf0, 0x9, 0x9}, {0xc068141140, 0x9, 0x9}, {0xc0c6de6240, ...}, ...)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/tae/dataio/blockio/read.go:68 +0x4a5
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*blockReader).Read(0xc1e6ad7340, {0x2cc5f58?, 0xc1d1e29c20?}, {0xc1643e4cf0?, 0x9?, 0x9?}, 0x0?, 0x0?)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/disttae/reader.go:83 +0x399
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*mergeReader).Read(0xc0b16c8618, {0x2cc5f58, 0xc1d1e29c20}, {0xc1643e4cf0, 0x9, 0x9}, 0xc098b08ea8?, 0x41ea47?)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/engine/disttae/reader.go:177 +0xd3
github.com/matrixorigin/matrixone/pkg/vm/pipeline.(*Pipeline).Run(0xc0032b7f28, {0x2ca5a28, 0xc0b16c8618}, 0xc0b2909c00)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/vm/pipeline/pipeline.go:70 +0x18b
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).Run(0xc0bc08ec30, 0xc091fde540?)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:69 +0x14e
github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun.func1(0x42b0460?)
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:89 +0x75
created by github.com/matrixorigin/matrixone/pkg/sql/compile.(*Scope).MergeRun
        /data1/action-runner/_work/mo-nightly-regression/mo-nightly-regression/head/pkg/sql/compile/scope.go:84 +0x20d

**Comment by LeftHandCold** [comment]
<img width="992" alt="image" src="https://user-images.githubusercontent.com/14086886/221818531-ae8b153b-a6e1-4bc0-a17f-71ff29612b83.png">
Please help @JackTan25  to check

**Comment by fengttt** [comment]
@LeftHandCold this is a new bug, has nothing to do with original bug.   Please create a new bug for this panic.   Original bug is still S-1.

**Comment by JackTan25** [comment]
@aressu1985 fix this in #8191

**Comment by aressu1985** [comment]
has been fixed : 5fbb8d0f66167956c6dc4fc57f94b1900f89608c
